src/blockrelay/compactblock.cpp

```diff
@@ -420,7 +420,7 @@ bool CompactReRequest::HandleMessage(CDataStream &vRecv, CNode *pfrom)
     }
     else
     {
-        if (hdr->nHeight < (chainActive.Tip()->nHeight - DEFAULT_BLOCKS_FROM_TIP))
```

Comment:
Looking at the code, I think DEFAULT_BLOCKS_FROM_TIP can be removed now that you've replaced it in these if clauses

Reply:
it is still used elsewhere...the original use is for testing when
IsChainNearlySyncd() and then we co-opted it for use within thintype
block which we probably shouldn't have.  Long story short, we still need
DEFAULT_BLOCKS_FROM_TIP.
…